src/views/AnnualReport.vue

```diff
@@ -1,45 +1,41 @@
 <template>
   <div id="annual-report">
     <confirm-dialog
-      ref="confirm"
       attach="#annual-report"
+      ref="confirm"
     />
 
     <fetch-error-dialog
+      attach="#annual-report"
       :dialog="fetchErrorDialog"
       @exit="navigateToDashboard(true)"
-      attach="#annual-report"
     />
 
     <resume-error-dialog
+      attach="#annual-report"
       :dialog="resumeErrorDialog"
       @exit="navigateToDashboard(true)"
-      attach="#annual-report"
     />
 
     <save-error-dialog
-      filing="Annual Report"
+      attach="#annual-report"
+      filingName="Annual Report"
       :dialog="saveErrorDialog"
       :disableRetry="busySaving"
       :errors="saveErrors"
       :warnings="saveWarnings"
       @exit="navigateToDashboard(true)"
       @retry="onClickFilePay()"
       @okay="resetErrors()"
-      attach="#annual-report"
     />
 
     <payment-error-dialog
-      :dialog="paymentErrorDialog"
-      @exit="navigateToDashboard(true)"
       attach="#annual-report"
-    />
-
-    <bcol-error-dialog
```

Comment:
Payment Error Dialog now handles BCOL pay errors or any other pay error, so bcol-error-dialog is deleted.

Reply:
Note also that `bcolObject` is replaced by `saveErrors` and `saveWarnings`.